### PR TITLE
add git secrets ci ros1 

### PR DESCRIPTION
**Pull request by ojasjoshi**
*Issue #, if available:*

*Description of changes:*
Adding git secrets scan action to ROS1 workflows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ojasjoshi** [comment]
Blocked by aws-robotics/aws-robomaker-github-actions#17.

Before merging, requires version bump (to 2.3.4) for all aws-robomaker-gh-actions after is merged.

File: .github/workflows/ros1.yml
Comment (mm318) [inline .github/workflows/ros1.yml:45]: Same comments as those in https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/pull/94.

File: test.sh
Comment (mm318) [inline test.sh]: I don't think this file should be here.

**Comment by mm318** [comment]
The comments in https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/pull/94 also apply here.

File: .github/workflows/ros1.yml
Comment (mm318) [inline .github/workflows/ros1.yml]: You had a concern that doing `--scan-history` would take a long time, right?

Anyway, I think we should move the scan to a separate workflow YAML file, so that the scanning can take place in parallel with building and bundling.

Comment (ojasjoshi) [reply .github/workflows/ros1.yml:45]: Yes I did.. but I tested --scan-history on some public repos with larger commit history and the time doesn't seem to exceed more than a minute at the most. I feel that should be considerably lesser than what build and bundle takes.

Comment (ojasjoshi) [reply .github/workflows/ros1.yml]: Yes I did.. but I tested --scan-history on some public repos with signicantly larger commit history and the time doesn't seem to exceed more than a minute at the most. I feel that should be considerably lesser than what build and bundle takes.

**Review by mm318 — APPROVED** [review]

**Pull request merged**
